sdk/core/core-rest-pipeline/test/node/formDataPolicy.spec.ts

```diff
@@ -30,7 +34,7 @@ describe("formDataPolicy (node-only)", function () {
     assert.ok(isBlob(parts[0].body));
 
     const buffers: Buffer[] = [];
-    for await (const part of (parts[0].body as Blob).stream() as ReadableStream<Uint8Array>) {
+    for await (const part of getRawContent(parts[0].body as Blob) as NodeJS.ReadableStream) {
```

Comment:
As mentioned last time RLC has the generated type including `ReadableStream<Uint8Array> |  NodeJS.ReadableStream` and we expected these types should be supported in core in both Node and Browser envinronments. Will this pr fix that issue? If not do we have an issue to track that?